src/main/java/ch/uzh/soprafs22/groupmatcher/service/MatcherService.java

```diff
@@ -47,6 +47,7 @@ public class MatcherService {
 
     private AnswerRepository answerRepository;
 
+
```

Comment:
It's a bit silly but honestly it would be great if you could remove this extra line because then MatcherService.java wouldn't have any diffs / wouldn't be in this PR, which will prevent any hassle after merging the other PR (87) which does have diffs on this file